runtime/ms-rest-azure/index.d.ts

```diff
@@ -321,7 +321,7 @@ export class AzureEnvironment {
     sqlManagementEndpointUrl: 'https://management.core.usgovcloudapi.net:8443/',
     sqlServerHostnameSuffix: '.database.usgovcloudapi.net',
     galleryEndpointUrl: 'https://gallery.usgovcloudapi.net/',
-    activeDirectoryEndpointUrl: 'https://login.microsoftonline.com/',
+    activeDirectoryEndpointUrl: 'https://login-us.microsoftonline.com/',
```

Comment:
@milope this https://github.com/Azure/azure-sdk-for-node/issues/2262 should now be fixed.